src/internal/Observable.ts

```diff
@@ -364,6 +367,7 @@ export class Observable<T> implements Subscribable<T> {
    * rejects on an error. If there were no emissions, Promise
    * resolves with undefined.
    */
+  /** @deprecated Deprecated use {@link firstValueFrom} or {@link lastValueFrom} instead */
```

Comment:
Hi there. This note is added in the new comment block. Is this intended? I'm not sure, but this may hide the first comment block for the documentation generator. Maybe this `@deprecated` part should be moved to the first comment block?

Reply:
Fixed!